src/AppBundle/Command/AddUserCommand.php

```diff
@@ -139,7 +140,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
         $password = $input->getArgument('password');
         if (null === $password) {
             $question = new Question(' > <info>Password</info> (your type will be hidden): ');
-            $question->setValidator(array($this, 'passwordValidator'));
+            $question->setValidator($this->getValidatorForProperty('plainPassword'));
```

Comment:
yes :)  but in my previous comment I was referring to validators defined inside the command. However, maybe doing that it's a wrong practice. That's why I want to read more opinions.